generic/piranha/test-resources/kt/feature_flag_system_1/control/expected/XPFlagCleanerCases.kt

```diff
@@ -0,0 +1,178 @@
+package com.uber.input
+
+import java.lang.annotation.Retention
+import java.lang.annotation.RetentionPolicy
+
+internal class XPFlagCleanerPositiveCases {
+    enum class TestExperimentName {
+
+
+    }
+
+    enum class AnotherTestExperimentName {
+
+
+    }
+
+    @Retention(RetentionPolicy.RUNTIME)
+    annotation class Autorollout(val staged: Boolean = false)
+
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
+    internal annotation class ToggleTesting(val treated: Array<TestExperimentName>)
+
+    private val experimentation: XPTest? = null
+    private var tBool = false
+    fun conditional_contains_stale_flag() {
+
+
+    }
+
+    fun conditional_with_else_contains_stale_flag() {
+        println("Hi world")
+    }
+
+     fun complex_conditional_contains_stale_flag() {
+        println("Hello World")
+
+    }
+
+    fun other_api_stale_flag() {
+        println("Hi world")
+    }
+
+    fun assignments_containing_stale_flag() {
+
+        tBool = false
+
+
+        tBool = false
+
+
+        tBool = true
+
+
+        tBool = tBool
+
+
+        tBool = false
+    }
+
+    fun return_contains_stale_flag(): Boolean {
+        // FIXME:
+        return false
+    }
+
+    fun condexp_contains_stale_flag() {
+
+        tBool = false
+    }
+
+    fun misc_xp_apis_containing_stale_flag() {
+
+
+
+
+
+
+
+
+
+
+
+
+
+        if ((tBool || true)) {
```

Comment:
are these blank lines really needed?

Reply:
They are not. The test compares the files by ignoring the whitespace. Updated.